client/rpc-core/debug/src/lib.rs

```diff
@@ -30,6 +30,7 @@ pub struct TraceParams {
 	pub disable_stack: Option<bool>,
 	/// Javascript tracer (we just check if it's Blockscout tracer string)
 	pub tracer: Option<String>,
+	pub tracer_config: Option<single::TraceCallConfig>,
```

Comment:
This could be a simple flag `pub with_log: Option<bool>`

Reply:
Is there a reason for creating that new type? `TraceCallConfig`

Reply:
nevermind, just checked the spec. It `withLog` is inside `tracerConfig`.

Reply:
@RomarQ it's because we nedd to comply with ethereum RPC specifications, so we can't choose the name of the field, it's expected to be a `tracer_config` field with an object with several subfields, we only support with_logs for the moment, but there is more subfields in the spec